packages/client/playwright.config.ts

```diff
@@ -12,21 +12,28 @@ const config: PlaywrightTestConfig = {
       use: {
         ...devices["Desktop Chrome"],
         //Artifacts
-      screenshot: "only-on-failure",
-      video: "retain-on-failure",
-      trace: "retain-on-failure",
-      headless: true,
+        screenshot: "only-on-failure",
+        video: "retain-on-failure",
+        trace: "retain-on-failure",
+        headless: true,
       },
     },
     {
       name: "openfin",
       use: {
-      screenshot: "only-on-failure",
-      video: "retain-on-failure",
-      trace: "retain-on-failure",
+        screenshot: "only-on-failure",
+        video: "retain-on-failure",
+        trace: "retain-on-failure",
       },
     },
   ],
+  /* Run your local dev server before starting the tests */
+  webServer: {
+    command: 'npm run openfin:dev',
+    port:1917,
```

Comment:
why do we need to specify port here?

Reply:
We don't really need to have this command as part of this PR.  This is something that Chuck and added when we attempted to spin up openfin within the tests framework.  I suggest to remove it from this PR scope

Reply:
yep, otherwise the rest of us are left wondering what the heck it does 😂

Reply:
🤝  removed 👍